evaluator/autoware_planning_evaluator/schema/autoware_planning_evaluator.schema.json

```
@@ -0,0 +1,98 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Autoware Planning Evaluator Nodes",
+  "type": "object",
+  "definitions": {
+    "autoware_planning_evaluator": {
+      "type": "object",
+      "properties": {
+        "output_file": {
+          "description": "output metrics file path",
+          "type": "string",
+          "default": ""
+        },
+        "ego_frame": {
+          "description": "reference frame of ego",
+          "type": "string",
+          "default": "base_link"
+        },
+        "selected_metrics": {
+          "description": "metrics to collect/record",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [
+            "curvature",
+            "point_interval",
+            "relative_angle",
+            "length",
+            "duration",
+            "velocity",
+            "acceleration",
+            "jerk",
+            "lateral_deviation",
+            "yaw_deviation",
+            "velocity_deviation",
+            "stability",
+            "stability_frechet",
+            "obstacle_distance",
+            "obstacle_ttc",
+            "modified_goal_longitudinal_deviation",
+            "modified_goal_lateral_deviation",
+            "modified_goal_yaw_deviation"
+          ]
+        },
+        "trajectory": {
+          "description": "trajectory object",
```

I think the [documentation rendering issue](https://autowarefoundation.github.io/autoware.universe/pr-8746/evaluator/autoware_planning_evaluator/) is caused by missing `type` fields.
```suggestion
          "description": "trajectory object",
          "type": "object",
```